tests/unit/test_timeseries.py

```diff
@@ -1,50 +0,0 @@
-import unittest
```

Comment:
Is this file empty now, or am I just confused by Github's rendering?

Reply:
I deleted this file and moved the tests into tests/unit/test_base.py next to the other TimeSeries tests.